### PR TITLE
fix pqrepair log4j setting

### PR DESCRIPTION
**Pull request by kaisecheng**
Fixed: #13725

## Release notes

Fix `pqrepair` unable to output log message

## What does this PR do?

This PR turns on `info` log level and add log4j properties to test classpath to enable log.
Also, add log message to let users know the status of repair.

## Why is it important/What is the impact to the user?

`pqrepair` does not print any log during the process, which is hard to trace PQ issue. 

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files (and/or docker env variables)
- [ ] I have added tests that prove my fix is effective or that my feature works

## Author's Checklist

- [ ]

## How to test this PR locally

you should see log in console by running the following command
$ bin/pqrepair /any/path/to/pq/dir

## Related issues

-

### CODE REVIEW

**Comment by kaisecheng** [comment]
CI failed due to missing 8.2.0-SNAPSHOT

**Review by andsel — CHANGES_REQUESTED** [review]
I've left a doubt to open too much the root logger, in root logger goes all he logs not captured by any logger, and so potentially could start to appear logs from libraries and we don't expect that. 
I haven't yet tested locally to check this, but maybe the best is to create a logger definition for the PQ tools specifically, and put that at `INFO` level.

File: logstash-core/src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -6,6 +6,6 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n
 
-rootLogger.level = error
+rootLogger.level = info
```
[the remark this answers is not in the document]
Comment (kaisecheng) [reply logstash-core/src/main/resources/log4j2.properties]: the default root log level is "info" in the [project](https://github.com/elastic/logstash/blob/v8.0.0/config/log4j2.properties#L61), right? Isn't it a problem now showing libraries log? Or the customized logger has done some magic that I miss?
My doubt is which parts depend on "logstash-core/src/main/resources/log4j2.properties", except pqcheck and pqrepair, and whether "error" level is a mistake from the initial commit or a design.

I added pqrepair logger in config and set log level back to "error". Can you give an example how "info" affects user log?

[the remark this answers is not in the document]
Comment (andsel) [reply logstash-core/src/main/resources/log4j2.properties]: Your are right, I totally missed the default INFO in `config/log4j2.properties`. My concerns was to not open too much the loggers, so the moving from `error` to `info`, but on Logstash distribution we are already at `info`. 
So no more concerns.

**Review by andsel — APPROVED** [review]
LGTM!
Thank's to to have added the `logger.pqrepair` for the PQRepair tool.

**Pull request merged**